### PR TITLE
New create ux

### PR DESCRIPTION
**Pull request by JakeHartnell**
Did this on the plane! Added a markdown editor too.

### CODE REVIEW

File: components/ProposalCard.tsx [hunk 1]
```diff
@@ -87,15 +87,13 @@ export default function ProposalCard({
   status,
   expires_at,
 }: ProposalCardProps) {
-  const expiresAtDateTime = new Date(expires_at / 1000000).toLocaleString()
+  //// TODO fix, expires_at time
```
Comment (gavindoughtie) [inline components/ProposalCard.tsx:90]: I use this convention for todos: `TODO(user.name): message` which is nice for grepping with a lot of contributors.

**Review by gavindoughtie — APPROVED** [review]
Code LGTM.

This is probably better for quick demos in Lisbon, though I'm concerned about the drivability of the UI as things get longer/larger. Also sad to see the fancy JSON editor gone. Do you think we should bring it back for v1?

File: components/CustomEditor.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}: {
+  dispatch: (action: ProposalAction) => void
+  customMsg: MessageMapEntry
+}) {
+  const [message, setMessage] = useState(JSON.stringify(customMsg.message))
+  const [error, setError] = useState(undefined)
+
+  function updateCustom() {
+    try {
+      const id = customMsg?.id ?? ''
```
Comment (gavindoughtie) [inline components/CustomEditor.tsx:22]: You want this to be `message?.id` don't you?

File: components/CustomEditor.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    } catch (err) {
+      console.error(err)
+    }
+  }
+
+  // Handles values from react-json-editor-ajrm
+  function handleMessage(msg) {
+    if (!msg.error) {
+      setError(undefined)
+      setMessage(msg.json)
```
Comment (gavindoughtie) [inline components/CustomEditor.tsx:49]: I think this is in the incorrect sequence and makes the proposal update lag behind

**Comment by JakeHartnell** [comment]
> Code LGTM.
> 
> This is probably better for quick demos in Lisbon, though I'm concerned about the drivability of the UI as things get longer/larger. Also sad to see the fancy JSON editor gone. Do you think we should bring it back for v1?

Can bring it back if we get it working nicely.

File: components/ProposalCard.tsx (hunk 1 above)
Comment (JakeHartnell) [reply components/ProposalCard.tsx]: Added a link to the new git issue I just created for this.

**Pull request merged**